WebApplications/src/test/java/edu/stanford/hivdb/graphql/SierraSchemaTest.java

```diff
@@ -41,8 +42,8 @@
 	@Test
 	public void testSequenceAnalysisDataFetcher() {
 		Map<String, Object> arguments = new LinkedHashMap<>();
-		List<Map<String, String>> sequences =
-			TestSequencesFiles.getTestSequences(TestSequencesProperties.PROBLEM_SEQUENCES)
+		List<Map<String, String>> sequences =	
+			FastaUtils.readStream(TestSequencesFiles.class.getClassLoader().getResourceAsStream(TestSequencesProperties.PROBLEM_SEQUENCES.propertyName))
```

Comment:
This should work right?

```java
List<Map<String, String>> sequences = FastaUtils.readStream(
    TestSequencesFiles.getTestSequenceInputStream(TestSequencesProperties.PROBLEM_SEQUENCES)
);
```

Reply:
Good call on that TestSequencesFiles class method. Updated.